addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -416,6 +425,11 @@ var SlideUploadDialog = Dialog.extend({
         var preventOnchange = $input.data('preventOnchange');
 
         var file = ev.target.files[0];
+        if (!file) {
+            this.$('#slide-image').attr('src', '/website_slides/static/src/img/document.png');
```

Comment:
If you don't want to see the previously loaded image while the new one is loading, just do: 
`this.$('#slide-image').attr('src', '');`
That way it won't flicker if the new file to load is small, and you won't see any 'fake' image if the new file to load is big.

Reply:
I did this this way because in 13.0 the certification form is different form the other ones and has a 'default' image shown. so I'm putting the default image back as long as no other image is loaded. Should I use `this.$('#slide-image').attr('src', '');`, it would show the broken file symbole. For the other forms, the image is hidden as the column is hidden and `#slide-image` is only used for display. In master, however I think we could do `this.$('#slide-image').attr('src', '');` as certification uses the the previewColumn to.

Reply:
Ah nope, my bad, the broken file icon is even worse.